Reportedly/AppDelegate.swift

```diff
@@ -29,24 +29,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         return true
     }
-
+    
```

Comment:
Do we have spaces here instead of tabs? Is this the convention? 🤔

Reply:
oh I use the cmd+I command for the entire file and this is how Xcode indents it, this is definitely Xcode convention, unless we have our custom ones, but for this project I think we should go with Apple Xcode default's settings 🙏